test/config/__snapshots__/env.spec.js.snap

```diff
@@ -1,5 +1,19 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`config/env .getConfig(env) supports Azure DevOps 1`] = `
+Object {
+  "endpoint": "an Azure DevOps endpoint",
+  "hostRules": Array [
+    Object {
+      "endpoint": "an Azure DevOps endpoint",
+      "platform": "vsts",
+      "token": "an Azure DevOps token",
+    },
+  ],
+  "platform": "vsts",
```

Comment:
migrate

Reply:
To change an auto generated file I changed a bit more files than I was planning. But I did what I could not to change much code, because it was said in the task  just to change doc files and tags.